downstream/snippets/snip-gw-authentication-common-checkboxes.adoc

```diff
@@ -0,0 +1,3 @@
+. To automatically create organizations, users, and teams upon successful login, select *Create objects*.
+. To enable this authentication method upon creation, select *Enabled*.
+. To remove a user for any groups they were previously added to when they  authenticate from this source, select *Remove users*.
```

Comment:
I think I see extra spaces between 'they authenticate', please check and update as necessary.